src/EditorFeatures/Test2/InlineHints/CSharpInlineParameterNameHintsTests.vb

```diff
@@ -398,5 +398,255 @@ class Derived : Base
 
             Await VerifyParamHints(input)
         End Function
+
+        <WorkItem(47597, "https://github.com/dotnet/roslyn/issues/47597")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.InlineParameterNameHints)>
+        Public Async Function TestNotOnEnableDisableBoolean1() As Task
+            Dim input =
+            <Workspace>
+                <Project Language="C#" CommonReferences="true">
+                    <Document>
+class A
+{
+    void EnableLogging(bool value)
```

Comment:
A test for `EnableLogging(bool value, string message)` might be good, if nothing else than i had to read through the whole thing before I saw the bit where it only applies to the first parameter 😛